Makefile

```diff
@@ -193,11 +193,13 @@ local-docker-rebuild:
 	docker-compose up -d quay-db --build
 	docker exec -it quay-db bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
 	docker-compose up -d quay --build
+	docker-compose restart quay
```

Comment:
Shouldn't `docker-compose up -d quay --build` start the build version automatically?

Reply:
Only if it detects a change. Some things, like the the entrypoint are mounted in and not detected on rebuild.